app/lib/features/chat/widgets/pending_req_list_view.dart

```diff
@@ -23,12 +22,6 @@ class PendingReqListView extends StatelessWidget {
         leading: ActerAvatar(
           mode: DisplayMode.User,
           uniqueId: userId,
-          avatarProviderFuture: avatar != null
```

Comment:
like here as well... Why is it gone without replacement? Some info in the PR description would have been good.

Reply:
Yeah, PR description is forgotten. Apologies, I'll add it right away.